examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -64,11 +75,28 @@ const ExternalDataView: React.FC<IExternalDataViewProps> = (props: IExternalData
 
                 const responseBody = await response.json() as Record<string, unknown>;
                 const newData = responseBody.taskList as TaskData;
-                if(newData !== undefined && !isEqual(newData,externalData)) {
+                if (newData !== undefined && !isEqual(newData, externalData)) {
                     console.log("APP: External data has changed. Updating local state with:\n", newData)
+                    const updatedExternalData = Object.entries(newData);
+                    let count = 0;
```

Comment:
This looks like you're trying to detect that the external data changed, but Kajari is already working on that part.  Perhaps just focus on detecting that the draft data has unsaved changes until Kajari's change is in